src/Infrastructure.EntityFramework/Repositories/OrganizationDomainRepository.cs

```diff
@@ -147,14 +147,13 @@ from s in sJoin.DefaultIfEmpty()
         using var scope = ServiceScopeFactory.CreateScope();
         var dbContext = GetDatabaseContext(scope);
 
-        //Get domains that have not been verified after 72 hours
-        var domains = await dbContext.OrganizationDomains
-            .Where(x => (DateTime.UtcNow - x.CreationDate).Days == 4
-                        && x.VerifiedDate == null)
+        var unverifiedDomainsInLast72Hours = await dbContext.OrganizationDomains
+            .Where(x => x.CreationDate.Date >= DateTime.UtcNow.AddDays(-4).Date
```

Comment:
Question: I'm not sure if a unit test would add value here since the behavior can only be tested at runtime.

Reply:
You can add an integration test on the project `Infrastructure.IntegrationTest` using the `DatabaseData` attribute which configures the test to run once for each configured DB provider.
Example here https://github.com/bitwarden/server/blob/main/test/Infrastructure.IntegrationTest/AdminConsole/Repositories/OrganizationRepositoryTests.cs